lib/lighthouse/benefits_education/service.rb

```diff
@@ -42,18 +44,29 @@ def inspect
       "#<#{self.class}:#{object_id} #{instance_variables_string}>"
     end
 
+    # REMOVE THIS - helper method just for QA purposes
+    def create_error_response(status)
+      headers = { 'content-type' => 'application/json' }
+      error_body = { status:, message: 'foo', error: 'bar', error_description: 'baz' }
+
+      response = OpenStruct.new(status:, headers:, body: OpenStruct.new(errors: [error_body]))
+
+      OpenStruct.new(response:)
+    end
+
     ##
     # Retrieve a veteran's Post-9/11 GI Bill Status
     # @return [String] A JSON string representing the veteran's GI Bill status.
     def get_gi_bill_status
-      response = begin
+      # TODO: test the FE response to different error statuses.  This code should be deleted.
```

Comment:
Is this TODO and the ones on lines 30 and 47 ones you want to remove before this PR is merged, or in a follow up PR?

Reply:
Yes sorry for the confusion but these TODOs are there in case any reviewer would like to manually exercise the code.